src/find/mod.rs

```diff
@@ -0,0 +1,476 @@
+mod matchers;
+
+use std::cell::RefCell;
+use std::error::Error;
+use std::io::{Write, stderr, stdout};
+use std::rc::Rc;
+use std::time::SystemTime;
+use walkdir::WalkDir;
+use walkdir::WalkDirIterator;
+
+pub struct Config {
```

Comment:
Sorry, my bad. Not `#derive[Default]` but `impl Default`. You can have both and in `impl Default` just call `new`. So used to write `#derive[...]`. :(

The benefit is that you can enable clients of `Config`, e.g., a struct having a `Config` as field, to be able to use `#derive[Default]`. https://doc.rust-lang.org/core/default/trait.Default.html#derivable

Another usage could be possible, say if I'm writing my configuration in a json file and then using serde to load that json file into some data structure like:

```rust
struct MyGigantConfig {
    #[serde(default)]
    findutils_config: findutils::..::Config
    // Some other configs
}
```
Which means just use the default value for `Config` if the json file doesn't specify it.